adapters/svc1zap/svc1zap_test.go

```diff
@@ -63,7 +63,8 @@ func TestSvc1ZapWrapper(t *testing.T) {
 	t.Run("caller origin and custom params", func(t *testing.T) {
 		buf := new(bytes.Buffer)
 		logger := svc1log.New(buf, wlog.DebugLevel)
-		logr2 := New(logger, WithOriginFromZapCaller(), WithNewParamFunc(prefixParamFunc)).WithOptions(zap.AddCaller())
```

Comment:
This `AddCaller` should be handled by svc1zap.New and not needed explicitly in the caller.